c2cgeoportal/scaffolds/create/CONST_print_url

```diff
@@ -1 +1 @@
-https://repo1.maven.org/maven2/org/mapfish/print/print-servlet/3.10.6/print-servlet-3.10.6.war
+https://repo1.maven.org/maven2/org/mapfish/print/print-servlet/3.10.7/print-servlet-3.10.7.war
```

Comment:
Shouldn't we add a changelog entry to say that we should get this file from the CONST_crate_template?

Reply:
I now added one - does it make sense like that?